src/pybind/mgr/cephadm/module.py

```diff
@@ -1463,6 +1463,22 @@ def remove_host(self, host):
 
         :param host: host name
         """
+        # Verify if it is possible to remove the host safely
+        daemons = [(d.daemon_type, str(d.daemon_id))
```

Comment:
NIT, but is it necessary to convert daemon_id to a str but not daemon_type? They are both type Optional[str].